src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/corporates/CorporateService.kt

```diff
@@ -15,6 +17,13 @@ class CorporateService(private val corporateRepository: CorporateRepository) {
       CorporateOrganisation(
         id = it.id,
         name = it.corporateName,
+        caseload = it.caseload?.let { caseload -> CodeDescription(caseload.id, caseload.description) },
```

Comment:
didn't fancy adding to https://github.com/ministryofjustice/hmpps-nomis-prisoner-api/blob/065e98982172a19791ce7a63226174cb0c49bbb6/src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/data/CodeDescription.kt then?

Reply:
good idea